api/transaction/routes.go

```diff
@@ -119,7 +119,7 @@ func SendTransaction(c *gin.Context) {
 		return
 	}
 
-	txHash, err := ef.SendTransaction(gtx.Nonce, gtx.Sender, gtx.Receiver, gtx.Value, gtx.Data, signature)
+	txHash, err := ef.SendTransaction(gtx.Nonce, gtx.Sender, gtx.Receiver, gtx.Value, 0, 0, gtx.Data, signature)
```

Comment:
change 0 to gtx.GasPrice and gtx.GasLimit

Reply:
Done, also removed some unused code